src/dumpContent.ts

```diff
@@ -60,12 +60,13 @@ async function dumpBinaries(data: unknown) {
 }
 
 function isBinaryAttribute(data: unknown): data is ['binary', { id: string }] {
+  if (!Array.isArray(data)) return false
```

Comment:
Strange, that TypeScript uses the following signature: `isArray(arg: any): arg is any[];`. I would have expected `arg is unknown[];`. Anyway your code works 😸.